comments-by-guideline-and-success-criterion.md

```diff
@@ -385,13 +385,26 @@ In WCAG 2, the Guidelines are provided for framing and understanding the success
 
 ###### Applying SC 2.1.1 Keyboard to Non-Web Documents and Software
 
-This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.1.1](https://www.w3.org/WAI/WCAG22/Understanding/keyboard#intent).
+Where ICT is or includes non-web software that can be run on a software platform that provides a device-independent keyboard interface service, this applies directly as written, and as described in [Intent from Understanding Success Criterion 2.1.1](https://www.w3.org/WAI/WCAG22/Understanding/keyboard#intent).
 
 <div class="note wcag2ict software">
 	
 Keyboard interface does not refer to a physical device but to the service of [platform software](#platform-software) (e.g. operating system, browser, etc.) that provides the software with keystrokes from any keyboard or keyboard substitute. When the [non-web software](#software) supports such a device-independent service of the platform software, and the non-web software functionality is made fully operable through the service, then this success criterion would be satisfied.</div>
 <div class="note wcag2ict software">
 
+**Proposal 1: Do not add any more notes.**</div>
+<div class="note wcag2ict software">
+	
+**Proposal 2: Note describing keyboard interface service** 
+	
+A "device-independent keyboard interface service" refers to the platform service that provides keystrokes to any software running on the platform. Inclusion of an on-screen keyboard can be done as well but does not meet this requirement since it does not allow for the use of keyboard alternatives.</div>
```

Comment:
```suggestion
A "device-independent keyboard interface service" refers to the platform service that provides keystrokes to any software running on the platform.</div>

<div class="note wcag2ict software">

Inclusion of an on-screen keyboard can be done as well but does not satisfy this requirement since it does not allow for the use of keyboard alternatives whereas support of input from the device-independent keyboard interface service does.</div>
```